autoextract_poet/items.py

```diff
@@ -2,26 +2,42 @@
 
 import attr
 
-from autoextract_poet.util import remove_unknown_fields
+from .util import split_in_unknown_and_known_fields
+
+
+class _ItemBase:
+    # Reserving an slot for _additional_attrs.
+    # This is done in a base class because otherwise attr.s won't pick it up
+    __slots__ = ("_additional_attrs", )
 
 
 @attr.s(auto_attribs=True, slots=True)
-class Item:
+class Item(_ItemBase):
+
+    def __attrs_post_init__(self):
+        self._additional_attrs = {}
 
     @classmethod
     def from_dict(cls, item: Optional[Dict]):
         """
-        Read an item from a dictionary, ignoring unknown attributes for
-        backwards compatibility
+        Read an item from a dictionary.
+
+        Unknown attributes are kept in the dict ``_additional_attrs``
+        so that they can be serialized also by the ``AutoExtractAdapter``.
```

Comment:
ItemAdapters don't handle serialization, they're allowing to access the underlying data in an unified way, and convert it to dicts, right? I wonder if we should change the wording here, and in other similar places.

Reply:
Addressed here: https://github.com/scrapinghub/autoextract-poet/pull/13/commits/bad3abc131bc33a84179d21075533d314e82414d